src/actions/blog/feed.cr

```diff
@@ -0,0 +1,31 @@
+class Blog::Feed < BrowserAction
+  get "/blog/feed.xml" do
+    posts = PostQuery.new.all[0...10]
```

Comment:
There is a cool way to use `first` with an arg that I think would be great to use here:

```suggestion
    posts = PostQuery.new.all.first(10)
```

Reply:
Ah I tried `.take(10)` but that didn't work so resorted to the range.

Reply:
Oh bummer! Thought it did but this is a-ok as-is